CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ changes in the following format: PR #1234***
 - Fix examiner site display (PR #8967)
 - bvl_feedback updates in real-time (PR #8966)
 - DoB and DoD format respected in candidate parameters (PR #9001)
+- Prevent indefinate "File to upload" fields from being added if files are browsed then cancelled (PR #9179)
```

Comment:
indefinite

Reply:
This is also confusing in the changelog because it doesn't say where / what module.

Reply:
Thanks! Fixed.